### ISSUE TITLE
Prefixing filenames with '@' in --attach leads to @ being included in attachement filename.

### ISSUE THREAD

**Issue by jen-Ya** [jetmore/swaks#55]
`*** DEPRECATION WARNING: Inferring a filename from the argument to --attach will be removed in the future.  Prefix filenames with '@' instead.`

Prefixing filenames with '@' in --attach leads to @ being included in attachement filename.

`/usr/bin/swaks --attach @test.txt ...`

<img width="210" alt="image" src="https://user-images.githubusercontent.com/2387053/223063409-d9747244-c47b-454e-b612-8c96ac52db08.png">

**Comment by jetmore** [comment]
```
mb 0 /Users/jetmore/Documents/git/swaks/testing/regressions > ../../swaks --attach @t.txt -t foo --dump --dump data | grep t.txt
Content-Type: application/octet-stream; name="@t.txt"
Content-Description: @t.txt
Content-Disposition: attachment; filename="@t.txt"
mb 0 /Users/jetmore/Documents/git/swaks/testing/regressions > ../../swaks --attach @./t.txt -t foo --dump --dump data | grep t.txt
Content-Type: application/octet-stream; name="t.txt"
Content-Description: t.txt
Content-Disposition: attachment; filename="t.txt"
```

**Comment by jetmore** [comment]
Thanks for reporting.  Fixed by #62, will be in the next release
